frontend/src/components/design-system/other/StepProgress.vue

```diff
@@ -1,13 +1,50 @@
 <script setup lang="ts">
+import { computed } from 'vue';
+
 export interface Props {
   current: number;
 }
 
-defineProps<Props>();
+const props = withDefaults(defineProps<Props>(), {
+  current: 1,
+});
+
+const classes = computed(() => ({
```

Comment:
Tohle bych vyhodil, spíš to mate.

Reply:
V tomhle případě je to asi vážně zbytečný. Mažu.